contracts/GovNftSugar.vy

```diff
@@ -44,6 +45,9 @@ struct GovNft:
 
 # Contracts / Interfaces
 
+interface IGovNFTFactory:
+  def govNFTs() -> DynArray[address, MAX_COLLECTIONS]: view
+
```

Comment:
might be useful to also use the `govNFTs(start, end)` function in case we have paginate